admin/class-expose-shortlinks.php

```diff
@@ -55,7 +55,7 @@ public function expose_shortlinks( $input ) {
 			$input[ $key ] = WPSEO_Shortlinker::get( $shortlink );
 		}
 
-		$input['default_query_args'] = WPSEO_Shortlinker::get_encoded_query();
```

Comment:
If `WPSEO_Shortlinker::get_encoded_query()` is no longer being used, why not remove it?